cmds/boot/netdiskboot/network.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"time"
+
+	"github.com/u-root/u-root/pkg/dhclient"
+	"github.com/vishvananda/netlink"
+)
+
+func configureDHCPNetwork() error {
```

Comment:
I saw this function multiple times now within the code base.

Reply:
You coded it in your branch, I just moved it to a different file :P

Reply:
Let me rephrase that - can you clean this up ? ;)

Reply:
I took a look over the different bootloaders and there is more to refactor possible in general.
I would suggest a refactoring of dhcp implementation across all bootloaders in a seperat PR to the main branch.